README.md

```diff
@@ -1,30 +1,63 @@
+Decentralized applications can use Mosaic to compute over a composed network of multiple blockchain systems in parallel.
 
-_While OpenST is available as-is for anyone to use, we caution that this is early stage software and under heavy ongoing development and improvement. Please report bugs and suggested improvements._
+Mosaic enables building scalable blockchain token economies through the bidirectional transposition of ERC20 tokens on one blockchain, the *origin* chain, and a utility token representation on another blockchain, the *auxiliary* chain.
 
-## OpenST Protocol
-The OpenST Protocol enables building scalable blockchain token economies through the bidirectional transposition of value tokens on one blockchain, the "value chain", and a utility token representation on another blockchain, the "utility chain".
+The Protocol defines a set of actions that are performed atomically across a gateway. A gateway for a given token is comprised of a `EIP20Gateway` contract on origin, a corresponding `EIP20CoGateway` contract on auxiliary, and and an ERC20 contract on auxiliary that mints and burns utility tokens for an equivalent value of ERC20 tokens staked and unstaked on origin. Atomicity is achieved by combining performance of the transpositional actions in a 2-phase-commit structure with a hash-timelock on each of the origin and auxiliary chains.
 
-In order to transpose value for utility, the Protocol defines a set of actions that are performed atomically across a gateway. A gateway for a given token is comprised of a gateway contract on the value chain, a corresponding co-gateway contract on a utility chain, and and an ERC20 contract on the utility chain that mints and burns utility tokens for an equivalent value of ERC20 tokens staked and unstaked on the value chain. Atomicity is achieved by combining performance of the transpositional actions in a 2-phase-commit structure with a hash-timelock on each of the value and utility chains.
+You can read [the draft of the mosaic whitepaper](https://github.com/OpenSTFoundation/mosaic-contracts/blob/develop/docs/mosaicv0.pdf) or [the original OpenST whitepaper](https://drive.google.com/file/d/0Bwgf8QuAEOb7Z2xIeUlLd21DSjQ/view).
 
-In addition to the gateway, the Protocol incorporates the concept of a `facilitator`. The facilitator relieves the end-user from the requirement to be online and act on multiple blockchains by staking a `bounty` in order to act on behalf of the user. This bounty is an economic incentive that ensures compliance with the Protocol.
+## Installation
 
-For more information on the fundamentals and mission of the Protocol, please consult the [whitepaper](https://drive.google.com/file/d/0Bwgf8QuAEOb7Z2xIeUlLd21DSjQ/view).
+```bash
+npm install @openstfoundation/mosaic-contracts
+```
+
+## Usage
+
+```js
+import mosaicContracts from '@openstfoundation/mosaic-contracts';
+
+const { gatewayAbi, gatewayBin } = mosaicContracts.EIP20Gateway;
```

Comment:
That object destructuring doesn't work, as the keys are `abi` and `bin`.

Reply:
🤦‍♂️ that's what I had before